README.md

```diff
@@ -1 +1,152 @@
+
+## 🛠 Built With <a name="html, css and JavaScript"></a>
+
+### Tech Stack <a name="html, css and JavaScript"></a>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Social_Media_links]**
+- **[navigation_bar]**
+- **[home_page]**
+- **[about_page]**
+- **[main_program_section]**
+- **[featured_artists_section]**
+- **[partners_section]**
+- **[footer]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://amen-musingarimi.github.io/musical-concert-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need any code editor, for example Visual Studio.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Run tests
+
+To run tests, run the following command: npx hint . and npx stylelint "\*_/_.{css,scss}" and npx eslint .
+
+### Deployment
+
+You can deploy this project using Github pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@Amen-Musingarimi](https://github.com/Amen-Musingarimi)
+- Twitter: [@Amen-Musingarimi](https://twitter.com/Amen-Musingarimi)
+- LinkedIn: [atmusingarimi](https://www.linkedin.com/in/atmusingarimi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[nav_bar]**
+- [ ] **[home_page]**
+- [ ] **[about_page]**
+- [ ] **[featured_artist_page]**
+- [ ] **[footer]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Amen-Musingarimi/musical-concert-capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partners, Lorna Kaboro and Kodjo Senou.
+
```

Comment:
- [ ] It would be nice if the "Acknowledgements" section gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :blush: